### ISSUE TITLE
Improve experience for editors of parent sections using the section listing page

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#1885]
## Problem

As a VAMC upgrade team member updating lots of VAMC systems in different sections at once, i don't have the tools to find my content easily. 

EG: URLs

## Hypothesis

Showing a contextual filter with child sections for whatever section i'm looking at instead of a listing that you drill down into will allow users who are editing content in multiple sections to find their content faster, and switch between sections more quickly. 

## AC / Implementation

- [ ] Validate our assumption with some VAMC upgrade team members, via Slack and/or a phone call.
- [ ] Existing list of child sections on the /section/* URLs is replaced with a filterable list 

## Assumption 

That this at least partially solves the problem uncovered in the satisfaction survey.

**Comment by davidmpickett** [comment]
@swirtSJW Yes. Documented example here from December:  https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11790

**Comment by davidmpickett** [comment]
Hypothesis: filter that says only show "content I have permission to edit" could be useful for any editor using the content view page in Drupal cms.va.gov/admin/content and would solve for people in multiple sections as well as anyone in a single section who is overwhlemed byt the content page
